x-pack/test/osquery_cypress/runner.ts

```diff
@@ -48,12 +49,13 @@ async function setupFleetAgent({ getService }: FtrProviderContext) {
   const policyEnrollmentKey = await createAgentPolicy(kbnClient, log, 'Default policy');
   const policyEnrollmentKeyTwo = await createAgentPolicy(kbnClient, log, 'Osquery policy');
 
-  await new AgentManager(policyEnrollmentKey, config.get('servers.fleetserver.port'), log).setup();
-  await new AgentManager(
-    policyEnrollmentKeyTwo,
-    config.get('servers.fleetserver.port'),
-    log
-  ).setup();
+  const port = config.get('servers.fleetserver.port');
+  const agent = await new AgentManager(policyEnrollmentKey, port, log).setup();
+  const hostname = agent.substring(0, 12);
+  await waitForHostToEnroll(kbnClient, hostname);
+  const agentTwo = await new AgentManager(policyEnrollmentKeyTwo, port, log).setup();
+  const hostnameTwo = agentTwo.substring(0, 12);
```

Comment:
can we return hostname from `AgentManager`, so we don't need to hardcode `(0, 12)`?